config/kubernetes/sig-docs/teams.yaml

```diff
@@ -156,25 +175,36 @@ teams:
     - tengqm
     - tfogo
     - xiangpengzhao
-    - zacharysarah
     - zhangxiaoyu-zidif
     - zparnold
     privacy: closed
   sig-docs-zh-owners:
-    description: ""
-    members:
+    description: Admins for Chinese content
+    maintainers:
     - hanjiayao
     - tengqm
+    members:
     - xiangpengzhao
     - zhangxiaoyu-zidif
     privacy: closed
   sig-docs-zh-reviews:
-    description: Reviews for Chinese docs PRs
+    description: PR reviews for Chinese content 
+    maintainers:
+    - tengqm
+    - xiangpengzhao
+    - zhangxiaoyu-zidif
     members:
     - chenrui333
     - idealhack
     - pigletfly
-    - tengqm
-    - xiangpengzhao
-    - zhangxiaoyu-zidif
     privacy: closed
+  website-milestone-maintainers:
+    description: Contributors who can use `/milestone` in the website repo. Defined in https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml.
+    members:
+    - sig-docs-en-owners
```

Comment:
Unfortunately, teams can't be nested like this :grimacing: 

You'll either need to add this as a "parent team" to the `sig-docs-*` teams like https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-cloud-provider/teams.yaml#L18 or expand all teams and list out members individually.

While the latter seems cumbersome, I'd mostly suggest that because GitHub says that (ref https://help.github.com/en/articles/about-teams#nested-teams):

> Child teams inherit the parent's access permissions, simplifying permissions management for large groups. Members of child teams also receive notifications when the parent team is @mentioned, simplifying communication with multiple groups of people.

I'm not sure if this is a cause for concern here but it _may_ lead to unexpected things with respect to permissions in the future.

Reply:
Understood! Listing folks out will be cumbersome for obvious reasons, but better safe than sorry.